docs/en/Architecture.md

```diff
@@ -39,7 +39,7 @@ client portion of Alluxio.
 ### Master
 
 Alluxio may be deployed in one of two master modes, [single master](Running-Alluxio-Locally.html) or
-[fault tolerant mode](Running-Alluxio-Fault-Tolerant-on-EC2.html). The master is primarily
+[fault tolerant mode](Running-Alluxio-Fault.html). The master is primarily
```

Comment:
@aaudiber , I have no idea why it is just like this as in my notebook, it is correct:

`MacBook-Air:docs caadecarvalho$ grep Fault en/Architecture.md`
`[fault tolerant mode](Running-Alluxio-Fault-Tolerant.html). The master is primarily`
`MacBook-Air:docs caadecarvalho$ git status en/Architecture.md` 
`On branch smallfix`
`nothing to commit, working directory clean`

Anyway, I'll fix it through the web.